.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+.DS_Store
```

Comment:
The insidious OS X .DS_Store file... 
No reason to have it in the repo.

Add ".DS_Store" to the local `.git/info/exclude` to keep it from being added without having to be listed in the .gitignore file.